lib/include/prjxray/xilinx/configuration.h

```diff
@@ -358,6 +359,19 @@ Configuration<ArchType>::InitWithPackets(const typename ArchType::Part& part,
 	return Configuration(part, frames);
 }
 
+template <typename ArchType>
+void Configuration<ArchType>::ExtractFrameAddresses(FILE* fp) {
```

Comment:
This method name is rather poor?  Should be `PrintFrameAddresses`?

Reply:
Renamed all auxiliary data relaterd methods from `Extract...` to `Print...`